components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -3415,6 +3458,49 @@ components:
           example: "85e3f4b8-0d22-4181-b1e3-1651f71b88bd"
       required:
         - id
+
+    UserRegistrantsList:
+      type: object
+      properties:
+        totalResults:
+          type: integer
+          description: "Number of user registrants."
+          example: 2
+        userRegistrants:
+          type: array
+          items:
+            $ref: '#/components/schemas/UserRegistrant'
+
+    UserRegistrant:
+      type: object
+      properties:
+        id:
+          type: string
+          example: "8we2f4b8-0d22-8234-l4n7-1682a91b12cn"
+        name:
+          type: string
+          example: "Username Password"
```

Comment:
Shall we add a more matching example

Reply:
Updated with the actual value that is returned from the backend.